src/components/AddItem.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
```

Comment:
What do you think about adding a space after this line? So libraries and what is created by you are separated; it's something I use to do to keep code cleaner and more organized :D